easybuild/easyconfigs/s/squashfs-tools/squashfs-tools-4.6.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'squashfs-tools'
+version = '4.6.1'
+
+homepage = ' https://github.com/plougher/squashfs-tools'
+description = """squashfs-fuse"""
```

can you put something more descriptive here?
```suggestion
description = """tools to create and extract Squashfs filesystems"""
```